jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+# Sitespeed job script
+# Assumes the following parameters set
+# as environment variables
+#
+# TEST_URL
+# EDX_USER
+# EDX_PASS
+# USE_BASIC_AUTH
+# AUTH_USER (optional, required if USE_BASIC_AUTH)
+# AUTH_PASS (optional, required if USE_BASIC_AUTH)
+# NUMBER_OF_TIMES
+# CONNECTION
```

Comment:
Should there be an error if any of these are missing? (or use defaults?) Could save a cycle.